src/main/java/rx/Completable.java

```diff
@@ -1971,19 +1972,40 @@ public final void subscribe(CompletableSubscriber s) {
         } catch (NullPointerException ex) {
             throw ex;
         } catch (Throwable ex) {
+            Exceptions.throwIfFatal(ex);
             ERROR_HANDLER.handleError(ex);
```

Comment:
I see from the javadocs of `Observable#unsafeSubscribe` (and from the code) that it should subscribe to an Observable and call `OnSubscribe` without error handling.